integration/testdata/schema/schema.bubbly

```diff
@@ -185,3 +185,38 @@ table "tempo_periods" {
         }
     }
 }
+
```

Comment:
I would also add a comment, describing what `.bubbly` test file this table relates to. Just for the reasons of consistency with the other tables in this file.